### PR TITLE
Updated mocks to the new model

### PR DESCRIPTION
**Pull request by ac-jorellanaf**
Updated mocks to the new model
Additionally:
- Some fixes to console warnings and displaying for the homepage badges
- Some renaming of model slots as appropriate

### CODE REVIEW

File: src/mocks/responses.js [hunk 1]
```diff
@@ -79,16 +79,16 @@ export const responses = {
   // Get Dataset details (embedded) Metadata Repository Service
   "GET /api/metldata/artifacts/embedded_public/classes/EmbeddedDataset/resources/GHGAD588887987": embeddedDataset[0],
   // Get Dataset details (embedded) Metadata Repository Service
-  "GET /api/metldata/artifacts/embedded_public/classes/EmbeddedDataset/resources/GHGAD588887988": embeddedDataset.map((x) => { let y = { ...x }; y.accession = "GHGAD588887988"; return y })[0],
+  "GET /api/metldata/artifacts/embedded_public/classes/EmbeddedDataset/resources/GHGAD588887988": embeddedDataset.map((x) => { let y = { ...x }; y.accession = "GHGAD588887988"; y.ega_accession = "EGAD588887988"; return y })[0],
```
Comment (Cito) [inline src/mocks/responses.js:82]:
```suggestion
  "GET /api/metldata/artifacts/embedded_public/classes/EmbeddedDataset/resources/GHGAD588887988": embeddedDataset.map((x) => { {...x, accession: "GHGAD588887988", ega_accession: "EGAD588887988"})[0],
```

Comment (Cito) [reply src/mocks/responses.js:82]: a bit simpler, similarly below.

Comment (Cito) [reply src/mocks/responses.js:82]: maybe better to add a function in data that returns the dataset for an accession, or prepare an object with accessions as keys

**Review by Cito — CHANGES_REQUESTED** [review]

File: src/mocks/responses.js (hunk 1 above)
Comment (ac-jorellanaf) [reply src/mocks/responses.js]: I am not sure if you can use the keys in the responses in the return object definition? I think I tried that before. In any case, I applied your suggestion

**Review by Cito — APPROVED** [review]

**Pull request merged**